sdk/storage/azure-storage-file/azure/storage/file/aio/file_client_async.py

```diff
@@ -770,11 +756,12 @@ def __init__(  # type: ignore
             raise ValueError("Unsupported method for encryption.")
 
         content_range = None
-        if start_range is not None:
-            if end_range is not None:
-                content_range = "bytes={0}-{1}".format(start_range, end_range)
+        if offset is not None:
+            if length is not None:
+                length = offset + length - 1  # Reformat to an inclusive range index
```

Comment:
Is there a reason we reuse length here? can we make it **offset_end = offset + length - 1**

Reply:
This one hasn't been addressed 😀

Reply:
Thanks @xiafu-msft - I'm merging these changes into the Stream PR now - will add this
FYI @rakshith91